mmv1/products/compute/terraform.yaml

```diff
@@ -1080,8 +1080,10 @@ overrides: !ruby/object:Overrides::ResourceOverrides
           network_name: "my-network"
           default_backend_service_name: "backend-default"
           hybrid_backend_service_name: "backend-hybrid"
+          internal_backend_service_name": "backend-internal"
```

Comment:
you have an extra quotation which is throwing the variable substitution off. Same for the other variable `internal_neg_name"`